content/en/internal_developer_portal/software_catalog/entity_model/entity_types.md

```diff
@@ -405,7 +405,7 @@ Learn more about [peer tags and inferred entities][4].
 
 You can define custom entity types beyond service, system, datastore, queue, and API. Custom entities allow you to represent any component or resource that is important to your organization but does not fit into the standard categories.
 
-First, define the kinds you want to use with [this API][1]. Only entities of the kinds you've explicitly setup are accepted. In the example below, a user is declaring a library with links, tags, and owning teams.
+First, define the kinds you want to use with [this API][1]. Only entities of the kinds you've explicitly setup are accepted. Once you've defined the allowed kinds, entities of that kind can be defined in the UI or programmatically sent through the existing [Software Catalog APIs][2], [Github integration][4], and [Terraform module][3]. In the example below, a user is declaring a library with links, tags, and owning teams. Custom entities can be send through the existing Software Catalog APIs, Github integration, and Terraform module.
```

Comment:
```suggestion
First, define the kinds you want to use with [this API][1]. Only entities of the kinds you've explicitly set up are accepted. After you've defined the allowed kinds, entities of that kind can be defined in the UI or programmatically sent through the existing [Software Catalog APIs][2], [GitHub integration][4], and [Terraform module][3]. In the example below, a user is declaring a library with links, tags, and owning teams.
```
Some minor readability edits. It also looked like the last sentence was a bit repetitive? I thought it was okay to remove it, but don't hesitate to add it back if it wasn't.